packages/core/src/api/traces/initialize.ts

```diff
@@ -1,18 +1,36 @@
 import type { Metas } from '../../metas';
-import { TransportItemType } from '../../transports';
 import type { Transports } from '../../transports';
-import type { TracesAPI } from './types';
-
-export function initializeTraces(transports: Transports, metas: Metas): TracesAPI {
-  const pushSpan: TracesAPI['pushSpan'] = (payload) => {
-    transports.execute({
-      type: TransportItemType.TRACE,
-      payload,
-      meta: metas.value,
-    });
+import type { GetActiveSpan, TracesAPI } from './types';
+
+export function initializeTraces(_transports: Transports, _metas: Metas): TracesAPI {
+  let tracer: unknown | null = null;
+  let getActiveSpanInternal: GetActiveSpan = () => {
+    throw new Error('Tracer is not initialized');
+  };
+
+  const getTracer: TracesAPI['getTracer'] = () => tracer;
+
+  const isInitialized: TracesAPI['isInitialized'] = () => tracer !== null;
+
+  const setTracer: TracesAPI['setTracer'] = (newTracer) => {
+    // TODO: add check if tracer is already set
+
+    tracer = newTracer;
+  };
+
+  const getActiveSpan: TracesAPI['getActiveSpan'] = () => {
```

Comment:
below does the thing as well:
```javascript
import { context, trace } from '@opentelemetry/api';
const activeSpan = trace.getSpan(context.active());
```

Imho we can rely on `@opentelemetry/api` for getting trace provider, tracer or span